.travis.yml

```diff
@@ -53,8 +73,16 @@ after_success:
         export TAG=latest;
         echo "TRAVIS_BRANCH=$TRAVIS_BRANCH, REPO=$REPO, PR=$PR, BRANCH=$BRANCH, TAG=$TAG";
         docker login -u $DOCKER_USER -p $DOCKER_PASS;
-        docker build -f docker/standalone/Dockerfile -t $REPO:$COMMIT docker/standalone/;
+        travis_wait 20 docker build -f docker/standalone/Dockerfile -t $REPO:$COMMIT docker/standalone/;
         docker tag $REPO:$COMMIT $REPO:$TAG;
         docker tag $REPO:$COMMIT $REPO:travis-$TRAVIS_BUILD_NUMBER;
         docker push $REPO;
     fi'
+  - 'if [ "$BRANCH" == "master" ]; then
+        export REPO=ubercherami/cherami-server;
+        export TAG=latest;
+        echo "TRAVIS_BRANCH=$TRAVIS_BRANCH, REPO=$REPO, PR=$PR, BRANCH=$BRANCH, TAG=$TAG";
+        travis_wait 20 docker build -f docker/server/Dockerfile -t $REPO:$COMMIT -t $REPO:$TAG -t $REPO:travis-$TRAVIS_BUILD_NUMBER docker/server/;
```

Comment:
can we use the same commands for the two builds for tagging? (either one is fine for me, just want to be consistent)